llvm/include/llvm/BinaryFormat/GOFF.h

```diff
@@ -169,6 +169,91 @@ enum SubsectionKind : uint8_t {
   SK_PPA1 = 2,
   SK_PPA2 = 4,
 };
+
+// The standard System/390 convention is to name the high-order (leftmost) bit
+// in a byte as bit zero. The Flags type helps to set bits in byte according
+// to this numeration order.
+class Flags {
+  uint8_t Val;
```

Comment:
`= 0` and make the default ctor defaulted `= default;`

Reply:
Changed.